charts/kubeclarity/values.yaml

```diff
@@ -341,3 +343,19 @@ kubeclarity-postgresql:
 
 # End of KubeClarity Postgres Values
 #######################################################################################
+
+#######################################################################################
+## KubeClarity External Postgres Values
+
+# notes: self deployed postgresql and external-postgresql can't be used at the same time
+# use this if you wan't to use an external postgresql database as your backend 
+kubeclarity-postgresql-external:
+  enabled: false
+  auth:
+    existingSecret: kubeclarity-postgresql-secret
```

Comment:
We use the key "existingSecret" for kubeclarity-postgresql because the secret we create for postgres is an existing secret from the sub-chart's perspective.

In the case of kubeclarity-postgresql-external, the secret isn't an existingSecret, we're creating it but that means you'll have to put your external DB password into the values.yaml which isn't ideal.

I think for external we might want two options:
 - use an existing secret (user should create this before doing the helm install so that helm never knows the password, this should be the recommended approach)
 - create a secret, use the template we already have and let the user provide their password in the values.yaml

For this PR if we want to leave the existing behaviour then I think we just need to change the field name to indicate that its not an existing secret but instead will be created.